src/BasketBundle/Block/BasketBlockService.php

```diff
@@ -22,7 +22,7 @@
 /**
  * @author Xavier Coureau <xcoureau@ekino.com>
  */
-class BasketBlockService extends BaseBlockService
+class BasketBlockService extends AbstractBlockService
```

Comment:
are you sure @jordisala1991 i think this is the basket block for the frontend users